superset/assets/javascripts/reduxUtils.js

```diff
@@ -64,13 +64,13 @@ export function addToArr(state, arrKey, obj) {
   return Object.assign({}, state, newState);
 }
 
-export function enhancer() {
-  let enhancerWithPersistState = compose(persistState());
+export function enhancer(persist = true) {
+  let enhancerWithPersistState = persist ? compose(persistState()) : compose();
```

Comment:
when persist is false as in the case of explorev2, we still compose enhancer with composer of redux devtool, just without the persistState() param, so I assume we we only need this param for persistState in our use cases so far?